grommet/grommet
# Removed HPE as a theme in core grommet

#### What does this PR do?

Removes HPE from the grommet base themes

#### Where should the reviewer start?

package.json and `src/themes`

#### What testing has been done on this PR?
manual
#### How should this be manually tested?
open storybook and make sure you can see the HPE theme in the All components story
#### Any background context you want to provide?

#### What are the relevant issues?
Fixes https://github.com/grommet/grommet/issues/2337
#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?

yes

#### Is this change backwards compatible or is it a breaking change?

breaking change. Now callers have to install `grommet-theme-hpe` and import it like:

```
import { hpe } from "grommet-theme-hpe";
```
